### ISSUE TITLE
HTMLField does not load correctly when model displayed as inline in Django Admin

### ISSUE THREAD

**Issue by tdsymonds** [divio/djangocms-text-ckeditor#382]
I'm having problems trying to get the HTMLField editor to work when the field is displayed on an inline. The two below screenshots demonstrates viewing the admin of the model directly, where it works, and then as an inline where it doesn't:

![standard](https://cloud.githubusercontent.com/assets/7289781/18993662/3b2024c2-871c-11e6-851c-4b2dc5d40ecd.JPG)
![inline](https://cloud.githubusercontent.com/assets/7289781/18993661/3b1b5cb2-871c-11e6-99c4-23f3b37f58e9.JPG)

models.py:

```python
from django.db import models
from djangocms_text_ckeditor.fields import HTMLField

class Header(models.Model):
    name = models.CharField(max_length=50)

    def __unicode__(self):
        return '%s' % self.name


class TestField(models.Model):
    header = models.ForeignKey('Header')
    text = HTMLField()

    def __unicode__(self):
        return self.text[:50]
```

admin.py:
```python
from django.contrib import admin
from .models import TestField, Header

@admin.register(TestField)
class TestFieldAdmin(admin.ModelAdmin):
    pass

class TestFieldInline(admin.StackedInline):
    model = TestField
    extra = 0


@admin.register(Header)
class HeaderAdmin(admin.ModelAdmin):
    inlines = (TestFieldInline,)
```

I'm using:
```
Django==1.9.10
django-cms==3.3.1
djangocms-text-ckeditor==3.1.0
```

Any ideas?!

Thanks

**Comment by czpython** [comment]
Hi there,
Please use the latest djangocms-text-ckeditor (3.2.1)

**Comment by tdsymonds** [comment]
Hi @czpython thanks for gettting back to me so quickly, I've tried that and still no luck unfortunately

**Comment by czpython** [comment]
@tdsymonds no problem.
Can you try installing the current rc? It's fully backwards compatible.

`pip install git+git://github.com/divio/djangocms-text-ckeditor#egg=djangocms-text-ckeditor`

Also make sure to restart server in case of pyc files and reset browser cache.

**Comment by tdsymonds** [comment]
Still no luck. Tried collecting static, clearing browser cache, restarting server etc.

To clarify, I'm now on:

```
Django==1.9.10
django-cms==3.4.0
djangocms-text-ckeditor==3.3.0rc1
```

**Comment by czpython** [comment]
Hey @tdsymonds, we can reproduce the issue locally.
We'll update you once a fix is out.
Thanks!

**Comment by czpython** [comment]
Small update, we were able to reproduce the issue on older versions of this app, as such we'll fix it after the coming release since we're on code freeze until Tuesday next week and this is not a regression.

**Comment by vxsx** [comment]
Should be fixed with 3.3.0
